src/templates/admin.html

```diff
@@ -117,36 +127,30 @@ <h1>Announcements</h1>
-                        <tr>
-                            <form method="POST">
-                                <input type="hidden" name="form_id" value="remove_announcement">
-                                <input type="hidden" name="announcement_id" value="{{ announcements[key].id }}">
-                                <td>{{ announcements[key].time.strftime("%H:%M") }}</td>
-                                <td>{{ announcements[key].text }}</td>
-                                {% if announcements[key].id != 'banana_time' %}
-                                    <td title="Remove announcement"><button class="btn btn-danger">Remove</button></td>
-                                {% else %}
-                                    <td title="Cannot remove banana time announcement"><button class="btn btn-secondary" disabled>Remove</button></td>
-                                {% endif %}
-                            </form>
+                        <tr id="announcement-id-{{ announcements[key].id }}" class="remove-announcement-form">
+                            <input type="hidden" name="announcement_id" value="{{ announcements[key].id }}">
+                            <td>{{ announcements[key].time.strftime("%H:%M") }}</td>
+                            <td>{{ announcements[key].text }}</td>
+                            {% if announcements[key].id != 'banana_time' %}
+                                <td title="Remove announcement"><button id="{{ announcements[key].id }}" class="btn btn-danger remove-announcement-button">Remove</button></td>
+                            {% else %}
+                                <td title="Cannot remove banana time announcement"><button class="btn btn-secondary" disabled>Remove</button></td>
+                            {% endif %}
                         </tr>
                     {% else %}
-                        <tr>
-                            <form method="POST">
-                                <input type="hidden" name="form_id" value="remove_announcement">
-                                <input type="hidden" name="announcement_id" value="{{ announcements[key].id }}">
-                                <td>{{ announcements[key].mins_before }} minutes before {{ announcements[key].banana_time.strftime("%H:%M") }}</td>
-                                <td>{{ announcements[key].text }}</td>
-                                <td><button class="btn btn-danger">Remove</button></td>
-                            </form>
+                        <tr id="announcement-id-{{ announcements[key].id }}">
+                            <input type="hidden" name="announcement_id" value="{{ announcements[key].id }}">
+                            <td>{{ announcements[key].mins_before }} minutes before banana time</td>
+                            <td>{{ announcements[key].text }}</td>
+                            <td><button id="{{ announcements[key].id }}" class="btn btn-danger remove-announcement-button">Remove</button></td>
```

Comment:
Consider adding the `title` attribute to the `<td>` element the same way you have done on line 143
```suggestion
                            <td title="Remove announcement"><button id="{{ announcements[key].id }}" class="btn btn-danger remove-announcement-button">Remove</button></td>
```